js/count_monkeys.js

```diff
@@ -0,0 +1,23 @@
+// https://www.codewars.com/kata/count-the-monkeys/train/javascript
+
+function monkeyCount(n) {
+  const array = [];
+  // solution with do while
+  // let i = 1;
+  // do{
+  //    array.push(i);
+  //    i++;
+  // } while (i < n + 1);
+
+  // solution with for loop
+  // for (let i = 1; i < n + 1 ; i++) {
+  //   array.push(i);
+  // }
+  // return array;
+
+  // solution with Array.from
+  return  Array.from({length : n}, (e, i ) => i + 1);
```

Comment:
I reckon that your prettier may be malfunctioning again 😄 Notice the space in `i )`

Reply:
btw. what's the reason for using single letter variable names? Consider using full variable names e.g. `element` and `index`. They are easier to read as words rather than single letters